ovn-tester/ovn_tester.py

```diff
@@ -196,13 +203,15 @@ def run_base_cluster_bringup(ovn, bringup_cfg):
             worker.ping_ports(ovn, ports)
 
 def run_test_density(ovn, cfg):
-    context_name = "density_heavy" if cfg.lb else "density_light"
-    ns = Namespace(ovn, f'ns_{context_name}')
+    context_name = "density_heavy" if cfg.n_vips else "density_light"
     with Context(context_name, cfg.n_pods) as ctx:
+        ns = Namespace(ovn, f'ns_{context_name}')
         for i in ctx:
             worker = ovn.select_worker_for_port()
             ports = worker.provision_ports(ovn, 1)
             ns.add_port(ports[0])
+            if cfg.n_vips and i <= cfg.n_vips:
```

Comment:
This is not exactly the same as our internal requirement for density-heavy, what we need is:
```
    create an address set AS-node-heavy
    for i in range(P):
        create pod p1
        create pod p2
        add VIP-i to cluster-wide load balancer with backend p2
```
With your code, you actually create `n_vips` pods and `n_vips` VIPs with one backend and *then* create another `n_pods - n_vips` pods without creating any VIP for them.